thirdparty/CMakeLists.txt

```diff
@@ -391,14 +383,28 @@ ExternalProject_Add(rocksdb
         -DWITH_TESTS=OFF
         -DWITH_GFLAGS=OFF
         -DUSE_RTTI=ON
-        -DWITH_OPENSSL=ON  # enable encryption
         -DCMAKE_BUILD_TYPE=Release
         -DWITH_JEMALLOC=${USE_JEMALLOC}
         -DJEMALLOC_ROOT_DIR=${TP_OUTPUT}
         -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
         -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-        -DPORTABLE=${ROCKSDB_PORTABLE}
-        )
+        -DPORTABLE=${ROCKSDB_PORTABLE})
+# Link error on MacOS, disable building encfs plugin.
+# See https://github.com/pegasus-kv/encfs/issues/4
+if (NOT APPLE)
+    set(ROCKSDB_OPTIONS
+            ${ROCKSDB_OPTIONS}
+            -DROCKSDB_PLUGINS=encfs)
+endif ()
+ExternalProject_Add(rocksdb
+        URL https://github.com/facebook/rocksdb/archive/refs/tags/v8.5.3.tar.gz
```

Comment:
`https://pegasus-thirdparty-package.oss-cn-beijing.aliyuncs.com/rocksdb-v8.5.3.tar.gz`.

The tar has been uploaded.